src/models/inventaris.ts

```diff
@@ -0,0 +1,7 @@
+export interface IAanduidingsobject {
```

Comment:
nu al effe een heads up: het model dat we van een interne api terugkrijgen (dus hetgeen uit de elastic search komt), verschilt van het model dat een GET op het id van het object teruggeeft. het ES model heeft veel minder data, en soms verschillen zelfs de veldnamen en de structuur. we hebben dus voor elk object een ES interface, een API interface, en dan het object model zoals wij het in de UI gaan gebruiken.

Reply:
waar ik naartoe wou: we gaan over de naamgeving moeten nadenken/ erover waken dat die consequent toegepast wordt.

Reply:
Ok, moet er voor nu iets gewijzigd worden?

Reply:
Momenteel heb ik die altijd exact overgenomen uit plannen :)

Reply:
ter ref:
https://github.com/OnroerendErfgoed/proces_beschermen/blob/develop/proces_frontend/src/interfaces/apiModel.ts#L230
https://github.com/OnroerendErfgoed/proces_beschermen/blob/develop/proces_frontend/src/models/aanduidingsobject.ts
meestal gebruiken we nu het api model voor zowel de ES resultaten als de api get resultaten. Maar in de code moet je er dan rekening mee houden dat de ES resultaten niet alle data hebben, en er dug een extra GET moet gebeuren.

we kunnen dat onderscheid duidelijker maken door extra api model (EsApi ofzo dunno) te maken, maar dat is dan extra werk/meer code. Morgen eens overleggen misschien met Yannick, Emrys & Pegah bij.

Reply:
Grote voorstander van een apart ES model te maken, ja